src/client/feed/Feed.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const Feed = ({ feed }) => {
+  return (
+    <div>
+      <h3>Search results</h3>
+      <div className="ui divided items">
+        {
+          feed.posts.map((feedItem) => {return (<FeedItem content={feedItem}/>);})
+        }
+      </div>
+    </div>
+  );
+};
+
+const FeedItem = ({ content }) => {
+  return (
+    <div className="item">
+      <div className="meta" style={{ minWidth: '40px', textAlign: 'center', verticalAlign: 'middle' }}>
+        {
+          content.source === 'twitter' ? <i className="twitter icon" /> : ''

```

Comment:
So you can't have a switch case in things like this :s

Any idea how we can solve having a different component based on the source?